### PR TITLE
Score card

### PR DESCRIPTION
**Pull request by LehiThomas**
- Added scoring logic to gatherCards
- Fixed errors in scoreCard

### CODE REVIEW

File: src/scoreCard.js [hunk 1]
```diff
@@ -1,29 +1,18 @@
+import createCard  from './createCard';
```
Comment (ToothpickFactory) [inline src/scoreCard.js]: Why is createCard imported here?

File: src/gatherCards.js [hunk 2]
```diff
@@ -20,7 +20,13 @@ function loopGather(maxFollow, maxPeople, resolve, reject, index = 0, cards = []
 		for(; index < scrolledPeople.length && cards.length < maxFollow; index++){
 			let card = createCard(scrolledPeople[index]);
 			// Score the card here
-			cards.push(card);
+			if (card.checkForFollowed || card.isProtected) {
```
Comment (ToothpickFactory) [inline src/gatherCards.js]: At some point we can swap this entire if statement out for a function that gets passed in. That will make gatherCards more versatile and work for more than just following.

File: src/index.js [hunk 3]
```diff
@@ -5,14 +5,14 @@ import followPeople from './followPeople';
 window.TwitterMinion =  (function(){
 	let myFollowing = 4997;
 	let myFollowers = 6238;
-	let maxFollow = 100 //assessMaxFollow(myFollowing, myFollowers);
+	let maxFollow = assessMaxFollow(myFollowing, myFollowers);
```
Comment (ToothpickFactory) [inline src/index.js]: Need to add a way of setting myFollowing and myFollwers

File: src/scoreCard.js (hunk 1 above)
Comment (LehiThomas) [inline src/scoreCard.js]: Oh I put that in there when i was trying to figure out why it wasn't working, thought maybe I need to import it... nope. I'll take it out and re-commit

**Pull request merged**